spec/index.html

```diff
@@ -661,6 +739,18 @@ <h2>Streams</h2>
     <p class="ednote">
       This is inconsistent with the processing description in <a href="#convert-stream" class="sectionRef"></a>.
     </p>
+    </section>
+
+  <section id="pre-loaded-contexts">
```

Comment:
Are those considerations specific to YAML? If not, I'd put there in another document. If yes, I think we are making things more complex than they should.

Reply:
There is general advice in the JSON-LD Best Practices document about caching contexts, and this falls under that guidance. This spec more strongly suggests that this particular context be implemented in the document loader. We can't update the core JSON-LD API spec for this right now.

If placed in a separate document (also non-normative, at this point), we'd need to add language here that implementations MUST adhere to those requirements, and we can't go back in time to require that JSON-LD processors adhere to the same requirements.

Reply:
> We can't update the core JSON-LD API spec for this right now.

Can't we schedule an update of that document (or just update the draft version, if any?)


> This spec more strongly suggests that this particular context be implemented in the document loader

So we need this only to support the convenience context?

Reply:
Issues surrounding accessing remote context continues to be a hot topic in JSON-LD (driven by schema.org) and should have a fair amount of discussion at TPAC, probably in the later portion of our F2F time.